docs/advanced_topics/accessibility_considerations.md

```diff
@@ -128,14 +128,14 @@ A number of built-in tools and additional resources are available to help create
 
 ### Built-in accessibility checker
 
-Wagtail includes an accessibility checker built into the [user bar](wagtailuserbar_tag). The checker can help authors create more accessible websites in compliance with the Web Content Accessibility Guidelines (WCAG) 2.0 and the Authoring Tool Accessibility Guidelines (ATAG) 2.0.
+Wagtail includes an accessibility checker built into the [user bar](wagtailuserbar_tag). The checker can help authors create more accessible websites following best practices and accessibility standards like [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/).
```

Comment:
- Generally "compliance" is reserved for legal compliance.
- WCAG 2.0 is relevant but also 2.1, soon 2.2, and more generally best practices
- The checker helps _Wagtail_ meet ATAG requirements, not the websites created with Wagtail